### DISCUSSION TITLE
v0.14.0-beta1

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#941] [Releases]
## ⚠️ Important change for dealing with time zones

As this is a beta release things could be unstable but I'm looking to fix the jank around dealing with time zones. To properly setup the beta release you can follow the updated steps here https://github.com/alexjustesen/speedtest-tracker/issues/935 as the docs are not yet updated.

## What's Changed
* [Docs] added star history to the readme by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/934
* [Bug] Refactor usage of time zones by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/929
* [Bug] Use admin url so intended destination is used by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/936
* Release v0.14.0-beta1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/940

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.13.4...v0.14.0-beta1

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.14.0-beta1'>v0.14.0-beta1</a>.</em>

**Comment by repcsi** [comment]
Hi Alex,

Should I update the docs with the info from #935 ?

**Reply by alexjustesen** [reply]
Not yet, going to update the docs when I release stable in case changes are needed.
